pkg/chains/chain_filters_test.go

```diff
@@ -48,6 +48,21 @@ func TestFilterChains(t *testing.T) {
 				return chains.ChainListByConsensus(chains.Consensus_solana_consensus, []chains.Chain{})
 			},
 		},
+		{
+			name: "Filter vm evm chains",
```

Comment:
Not directly related to this PR but test name should be full lower case if we can make the change in this file here